frontend/whiteboard-app/src/components/NotizBox.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react' ;
+import { FaTrash } from 'react-icons/fa';
+import Note from '../types';
+import '../index.css';
+
+const NotizBox = ({noteId , title , description} : Note ) => {
+    return (
+        <div  className='whiteBoardParent_NotizBox'>
```

Comment:
Du brauchst hier keine 2 Wrapper divs. Du kannst dem ersten einfach beide classNames mitgeben.

Reply:
Anke ich habe versucht nur 1 Div anzuwenden , aber leider das hat bei mir nicht geklappt , div war sehr breiter ,  ich habe 2 Div angewendet , damit ich in der Lage bin , mehr Kontrolle an div zu haben